src/components/IdeasGrid.js

```diff
@@ -77,54 +114,229 @@ const IdeasGrid = (props) => {
+              height: '100%',
+              backgroundColor: 'rgba(0, 0, 0, 0.5)',
+              zIndex: 1000,
+            }}
+            onClick={() => {
+              setShowImage(false);
+              setUrl('');
+            }}
+          />
+
+          {/* Modal Content */}
+          <div
+            ref={iframeContainerRef}
+            style={{
+              position: 'fixed',
+              top: '50%',
+              left: '50%',
+              transform: 'translate(-50%, -50%)',
+              width: window.innerWidth <= 768 ? '95%' : '90%',
+              maxWidth: '1200px',
+              maxHeight: '95vh',
+              backgroundColor: '#fff',
+              borderRadius: '12px',
+              boxShadow: '0 20px 60px rgba(0, 0, 0, 0.3)',
+              zIndex: 1001,
+              overflow: 'hidden',
+              display: 'flex',
+              flexDirection: 'column',
+            }}
+          >
+            {/* Header */}
+            <div
+              style={{
+                backgroundColor: '#f8f9fa',
+                padding: '16px 20px',
+                display: 'flex',
+                alignItems: 'center',
+                justifyContent: 'space-between',
+                borderBottom: '1px solid #dee2e6',
+                flexShrink: 0,
+              }}
+            >
+              <h3
+                style={{
+                  margin: 0,
+                  fontSize: '18px',
+                  color: '#495057',
+                  fontWeight: '600',
+                }}
+              >
+                Generated Character
+              </h3>
+              <button
+                onClick={() => {
+                  setShowImage(false);
+                  setUrl('');
+                }}
+                style={{
+                  background: '#495057',
+                  border: 'none',
+                  borderRadius: '6px',
+                  marginTop: '10px',
+                  width: '32px',
+                  height: '32px',
+                  color: 'white',
+                  cursor: 'pointer',
+                  fontSize: '18px',
+                  fontWeight: 'bold',
+                  display: 'flex',
+                  alignItems: 'center',
+                  justifyContent: 'center',
+                }}
+                onMouseOver={(e) => (e.target.style.background = '#c82333')}
+                onMouseOut={(e) => (e.target.style.background = '#495057')}
+              >
+                ×
+              </button>
+            </div>
+
+            {/* Content Area */}
+            <div
+              style={{
+                flex: 1,
+                overflow: 'auto',
+                padding: '20px',
+              }}
+            >
+              {/* Image */}
+              <div
+                style={{
+                  marginBottom: '24px',
+                  textAlign: 'center',
+                  width: '100%',
+                }}
+              >
+                <iframe
```

Comment:
have image fit the content height set so on desktop there are not navigation bars going sideways

<img width="1717" height="1032" alt="Screenshot 2025-10-14 at 5 29 27 PM" src="https://github.com/user-attachments/assets/22b4b74c-2aef-48a0-a751-071163c63824" />